docs/sphinx/user_guide/tutorial/scan.rst

```diff
@@ -18,23 +18,26 @@
 Parallel Scan Operations
 --------------------------------------------------
 
-Key RAJA features shown in this example:
+Key RAJA features shown in this section:
 
   * ``RAJA::inclusive_scan`` operation
   * ``RAJA::inclusive_scan_inplace`` operation
   * ``RAJA::exclusive_scan`` operation
   * ``RAJA::exclusive_scan_inplace`` operation
   * RAJA operators for different types of scans; e.g., plus, minimum, maximum, etc.
 
-In the examples, we demonstrate a variety of scan operations supported by RAJA.
-We show how different scan operations can be performed by passing different
-RAJA operators to the RAJA scan template methods. Each operator is a template
-type, where the template argument is the type of the values it operates on.
-For a summary of RAJA scan functionality, see :ref:`scan-label`. 
+In the following examples, we present examples of RAJA sequential, OpenMP,
```

Comment:
Maybe: "In the following, we present examples".

Reply:
Fixed.